frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -456,12 +475,153 @@ export const ObjectDetailProperties = {
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
     ...columnSettings({ hidden: true }),
+    // NOTE: table column settings may be identified by fieldRef (possible not normalized) or column name:
+    //   { name: "COLUMN_NAME", enabled: true }
+    //   { fieldRef: ["field", 2, {"source-field": 1}], enabled: true }
+    "detail.columns": {
+      section: t`Columns`,
+      title: t`Visible columns`,
+      widget: ChartSettingOrderedColumns,
+      getHidden: () => false,
+      isValid: ([{ card, data }]: any) =>
```

Comment:
The proper type here would be [`Series` (link)](https://github.com/metabase/metabase/blob/f5d85e8131903976d4b4194f7f5de728b522fe9c/frontend/src/metabase-types/types/Visualization.ts#L82)